client/landing/stepper/index.tsx

```diff
@@ -95,14 +91,17 @@ async function main() {
 	}
 
 	const flowName = getFlowFromURL();
-	const siteId = getSiteIdFromURL();
+	const flowLoader = availableFlows[ flowName ];
 
-	if ( ! flowName ) {
-		// Stop the boot process if we can't determine the flow, reducing the number of edge cases
-		return ( window.location.href = `/setup/${ DEFAULT_FLOW }${ window.location.search }` );
+	if ( ! flowLoader ) {
```

Comment:
Checking `flowLoader` instead of `flowName`, since if `flowName` is empty string or even `null` / `undefined`, `flowLoader` will be `undefined`.